sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_container_repository_client.py

```diff
@@ -194,6 +195,8 @@ async def set_tag_properties(self, tag: str, permissions: ContentPermissions, **
         :returns: None
         :raises: None
```

Comment:
It looks like these (and the return/raise types for `set_manifest_properties`) need an update like the sync client got